tests/flow_validation/test_action_state.py

```diff
@@ -0,0 +1,326 @@
+import typing as t
+
+import pytest
+
+from globus_automate_client.models import FlowDefinition, FlowValidationError
+
+simple_action_state = {
+    "StartAt": "ActionState",
+    "States": {
+        "ActionState": {
+            "Type": "Action",
+            "Comment": "No info",
+            "ActionUrl": "http://localhost:5000",
+            "Parameters": {},
+            "ResultPath": "$.some_path",
+            "End": True,
+        },
+    },
+}
+
+action_state_with_catcher = {
```

Comment:
I guess these which have a `Next` value need to be complete flows, so there's some opportunity to do what I suggested above, but perhaps not as many as I thought on my first skim through.

Reply:
Actually... since the `Next` values get validated at the `FlowDefinition` level, there is no validation happening at each state's level that ensures that `Next` points to something that exists. So I think your comment above makes sense and would greatly simplify the test cases. I'm going to push a commit with your style recommendations and then another commit where I change the state tests to only test the state-relevant fields.